src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.js

```diff
@@ -71,14 +75,21 @@ function Widget({ location, children, editor, insertAt }) {
 
 function Panel({ breakpoint, editor, insertAt }) {
   const [editing, setEditing] = useState(false);
+  const [width, setWidth] = useState(getEditorWidth(editor));
   const [inputToFocus, setInputToFocus] = useState("logValue");
 
   const toggleEditingOn = () => setEditing(true);
   const toggleEditingOff = () => setEditing(false);
+  const setEditorWidth = () => setWidth(getEditorWidth(editor));
+
+  useEffect(() => {
+    editor.editor.on("refresh", setEditorWidth);
```

Comment:
Ideally there'd be a resize event we can listen for since refresh gets noisy, but I don't think CodeMirror has that outside of refresh. This should be okay.